Formula/newrelic.rb

```diff
@@ -0,0 +1,29 @@
+class Newrelic < Formula
```

Comment:
We should rename this to `NewrelicCli` and change the file to `newrelic-cli` as well.  `newrelic` is probably too general at this time.

Reply:
Done.